src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -44,7 +51,7 @@ public String greeting(@RequestParam(name="name", required=false, defaultValue="
      * Method that find all the information of one entity.
      * @return: a collection type list with the information of book table
      */
-    @GetMapping
+    @GetMapping()
     public Iterable findAll() {
```

Comment:
you should document all or most of the methods

Reply:
I made documentantions to all methods